ote/ote/modules/arg_parsers/reid.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+
+from ote.api import (train_args_parser,
+                     test_args_parser,
+                     export_args_parser,
+                     compression_args_parser)
+
+from .default import DefaultArgParser
+from ..registry import ARG_PARSERS
+
+
+@ARG_PARSERS.register_module()
+class ReidArgParser(DefaultArgParser):
+    def get_train_parser(self, config_path):
+        parser = argparse.ArgumentParser(parents=[train_args_parser(config_path)], add_help=False)
+        parser = self._add_aux_weights_load_arg(parser)
+        parser = self._add_classes_arg(parser)
+        return parser
+
+    def get_test_parser(self, config_path):
+        parser = argparse.ArgumentParser(parents=[test_args_parser(config_path)], add_help=False)
+        parser = self._add_aux_weights_load_arg(parser)
+        parser = self._add_classes_arg(parser)
+        return parser
+
+    def get_export_parser(self, config_path):
+        parser = argparse.ArgumentParser(parents=[export_args_parser(config_path)], add_help=False)
+        return self._add_classes_arg(parser)
+
+    def get_compression_parser(self, config_path):
+        # Please, note that for image classification compression
+        # batch_size should be set from the compression config file,
+        # not from the command line arguments
+        # (smaller batch size often gives better results with "regularization by noise")
+        parser = argparse.ArgumentParser(parents=[compression_args_parser(config_path, with_batch_size=False)],
+                                         add_help=False)
+        parser = self._add_aux_weights_load_arg(parser)
+        parser = self._add_classes_arg(parser)
+        return parser
+
+    @staticmethod
+    def _add_classes_arg(parser):
+        parser.add_argument('--classes', required=False,
+                            help='Comma-separated list of classes (e.g. "cat,dog,mouse").')
+        return parser
+
+    @staticmethod
+    def _add_aux_weights_load_arg(parser):
+        parser.add_argument('--load-aux-weights', required=False,
```

Comment:
to make it work as the template you need to add this arg to extra args section like here https://github.com/openvinotoolkit/training_extensions/blob/develop/models/image_classification/model_templates/custom-classification/mobilenet_v3_small/template.yaml#L37

Reply:
Also such additional argument has to be added to ALL parers, but it might be ignored inside corresponding script.

Reply:
Done

Reply:
ALL is a keyword

Reply:
Sorry, Ilya, I do not understand you.

But I found out that I forgot to commit the reid parser with the latest change (this change added `--load-aux-weights` to export cmd line parser for reid)
Did you meant this?